authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+# Call Microsoft Graph from a Cloud Solution Provider application
+
+This topic describes how to enable application access to partner managed customer data via Microsoft Graph using either the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a>. 
+
+> **Note:** This topic applies **only** to Microsoft Cloud Solution Provider (CSP) application developers. The <a href="https://partner.microsoft.com/en-US/cloud-solution-provider" target="_newtab">Microsoft Cloud Solution Provider (CSP)</a> program enables Microsoft’s partners to resell and manage Microsoft Online services to customers. 
+
+**Important**: Calling Microsoft Graph from a CSP application is only supported for directory resources (such as **user**, **group**,**device**, **organization**) and [Intune](..\beta\resources\intune_graph_overview) resources.
```

Comment:
If we have two alerts in a row, they will lose emphasis. Consider moving the first note to come before the intro paragraph? Not ideal, but an option to separate the alerts. Also, for the Important, move the colon inside the **.